core/src/main/java/org/infinispan/transaction/xa/recovery/RecoveryManager.java

```diff
@@ -20,20 +20,25 @@
 public interface RecoveryManager {
 
    /**
-    * Returns the list of transactions in prepared state from both local and remote cluster nodes.
-    * Implementation can take advantage of several optimisations:
+    * Returns the list of transactions in prepared state from both local and remote cluster nodes. Implementation can
+    * take advantage of several optimisations:
     * <pre>
```

Comment:
I'd rather add a blank line before the second sentence and use `<ul>` instead of `<pre>`.